bitnami/mongodb-sharded/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb-sharded
-version: 1.2.0
+version: 1.2.1
```

Comment:
Could you increase this as a minor version (i.e. 1.3.0)? Even though it is fixing an issue, it is adding new functionality to the existing chart.

Reply:
sure — done